crates/consensus/beacon/src/engine/sync.rs

```diff
@@ -598,7 +599,9 @@ mod tests {
             header.parent_hash = hash;
             header.number += 1;
             header.timestamp += 1;
-            sealed_header = header.seal_slow();
+            let sealed = header.seal_slow();
+            let (header, seal) = sealed.into_parts();
+            sealed_header = SealedHeader::new(header, seal);
```

Comment:
pls open issue if one doesn't exist to remove `SealedHeader` in favour of alloy `Sealed` as you suggested, to avoid these conversions

Reply:
Done https://github.com/paradigmxyz/reth/issues/11123